Cargo.toml

```diff
@@ -1,24 +1,7 @@
-[package]
-name = "orml"
-version = "0.0.1"
-authors = ["Laminar Developers <hello@laminar.one>"]
-edition = "2018"
-
-[[bin]]
-name = "orml"
-path = "src/main.rs"
-
-[dependencies]
-oracle = { package = "orml-oracle", path = "oracle", default-features = false }
-tokens = { package = "orml-tokens", path = "tokens", default-features = false }
-traits = { package = "orml-traits", path = "traits", default-features = false }
-utilities = { package = "orml-utilities", path = "utilities", default-features = false }
-
-[features]
-default = ["std"]
-std = [
-	"oracle/std",
-	"tokens/std",
-	"traits/std",
-	"utilities/std",
+[workspace]
```

Comment:
I had some issue with `cargo check --all  --no-default-features` with this setup. Can you make sure the targets in Makefile works? Or if you can a better way to do check and run tests.

Reply:
@xlc Will resolve this err. I'm thinking of adding a example crate for both `orml` usage demo and integration check/tests purpose, we could probably check `--no-default-features` in the example crate.